data/examples/vm/values-istio-meshexpansion-gateways.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  values:
+    global:
+      # Provides dns resolution for global services
+      podDNSSearchNamespaces:
+        - global
```

Comment:
line 8 won't work (routing doesn't work properly for service.global).  Additionally DNS is limited to 6 entries, and we may overload the DNS cache.

Reply:
removed the whole section here and verifies it works